packages/react-ui/src/hatchifyReact/hatchifyReact.tsx

```diff
@@ -108,7 +108,11 @@ export function hatchifyReact<
           <HatchifyCollection<TSchemas, GetSchemaNames<TSchemas>>
             finalSchemas={finalSchemas}
             partialSchemas={partialSchemas}
-            schemaName={schema.name}
+            schemaName={
+              schema.namespace
```

Comment:
i know this _only_ duplicated once, but let's move it up to line ~105 and then pass in `schemaName={finalSchemaName}` or something (no idea what to name it lol)